packages/@okta/vuepress-site/docs/api/resources/authn/index.md

```diff
@@ -869,16 +869,18 @@ Content-Type: application/json
 
 #### Primary Authentication with Device Fingerprinting
 
-Authenticates a user via a [trusted application](#trusted-application) or proxy that overrides the [client request context](/docs/api/getting_started/design_principles#client-request-context).
+Include the `X-Device-Fingerprint` header to supply a device fingerprint. The device fingerprint is used in the following ways
+* If the new or unknown-device email notification is enabled, an email is sent to the user if the device fingerprint sent in the header is not associated with previous successful logins. 
+For more information about this feature, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?)
+* If you have the security behavior detection feature enabled and you have a new device behavior configured in a policy rule, a new device is detected if the device fingerprint sent 
+in the header is not associated with previous successful logins. For more information about this feature, see [EA documentation](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm?)
 
-Include the `X-Device-Fingerprint` header to supply a device fingerprint.
+Specifying your own device fingerprint is a highly privileged operation limited to trusted web applications and requires making authentication requests with a valid *API token*.
```

Comment:
We don't use italics in our documentation, so please remove this.

Reply:
ack